app/src/main/res/layout/fragment_registration.xml

```diff
@@ -163,13 +163,13 @@
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
                     android:digits="0123456789"
+                    android:inputType="numberPassword"
                     android:background="@drawable/white_rounded_background"
                     android:hint="@string/student_id"
-                    android:imeOptions="actionDone"
+                    android:imeOptions="flagNavigateNext"
```

Comment:
The next button cannot be used to go further. This works correctly in the first image, but not in the second
![photo_2023-12-20_16-54-02](https://github.com/NN171/Head-of-the-group-helper/assets/129608390/24af82da-710c-456e-86d4-09053d9e9cca)

![photo_2023-12-20_16-54-07](https://github.com/NN171/Head-of-the-group-helper/assets/129608390/5af07312-abe0-4f48-b6c6-e66801e4c00e)